src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -71,6 +71,13 @@ public synchronized void deleteTask(ReadOnlyTask target) throws TaskNotFoundExce
     }
 
     @Override
+    public synchronized ReadOnlyTask deleteTask(String target) throws TaskNotFoundException {
+        ReadOnlyTask capture = addressBook.getTask(target);
```

Comment:
See how the original source was getting tasks. Creating your own method here is a waste of time.